debian/control

```diff
@@ -1,26 +1,14 @@
 Source: io.elementary.camera
 Section: x11
 Maintainer: Cody Garver <cody@elementary.io>
-Build-Depends: appstream,
```

Comment:
Let's keep in the build-deps. Even though we don't need them to build the package, removing them breaks our gettext action and it's also convenient to be able to use `apt build-dep`